### ISSUE TITLE
Some nodes have a negative triangle count

### ISSUE THREAD

**Issue by gkjohnson** [gkjohnson/three-mesh-bvh#53]
@mquander 

I added a function to the benchmark utils to interrogate the structure of the tree a bit more called [getExtremes](https://github.com/gkjohnson/three-mesh-bvh/blob/master/benchmark/utils.js#L85). It returns the min and max depth and triangle count per node. When running it against the geometry in the benchmark script I'm seeing that the "minimum" number of triangles in a node is negative:

```js
const geometry = new THREE.TorusBufferGeometry( 5, 5, 700, 300 );
geometry.computeBoundsTree();

console.log( getExtremes( geometry.boundsTree ) );

// { depth: { min: 11, max: 22 }, tris: { min: -293, max: 304 } }
```

Maybe you have some thoughts?

**Comment by mquander** [comment]
No idea! I'm on vacation this next week, but I might find some time to look into it in a few days.

**Comment by gkjohnson** [comment]
Fixed in #54
